js/src/block-editor/index.js

```diff
@@ -11,15 +11,9 @@ import { setLocaleData } from '@wordpress/i18n';
  */
 import { addControls, registerBlocks } from './helpers';
 import { Edit } from './components';
-import { GAClient } from '../common/classes';
 
 setLocaleData( { '': {} }, 'genesis-custom-blocks' );
 addFilter( 'genesisCustomBlocks.controls', 'genesisCustomBlocks/addControls', addControls );
 
 // @ts-ignore
 registerBlocks( genesisCustomBlocks, gcbBlocks, Edit );
-
-// @ts-ignore
-window.GcbAnalytics = {
-	GAClient: new GAClient(),
```

Comment:
This isn't used anywhere. And I mistakenly kept in the copied `GA_ID` from Genesis Blocks.